syft/pkg/cataloger/common/cpe/cpe-index.json

```diff
@@ -0,0 +1,1293 @@
+      "webdriver-launcher": "cpe:2.3:a:webdriver-launcher_project:webdriver-launcher:*:*:*:*:*:node.js:*:*",
+      "webpack-subresource-integrity": "cpe:2.3:a:webpack-subresource-integrity_project:webpack-subresource-integrity:*:*:*:*:*:node.js:*:*",
+      "webrtc-native": "cpe:2.3:a:webrtc:webrtc-native:*:*:*:*:*:node.js:*:*",
+      "welcomyzt": "cpe:2.3:a:welcomyzt_project:welcomyzt:*:*:*:*:*:node.js:*:*",
+      "wffserve": "cpe:2.3:a:wffserve_project:wffserve:*:*:*:*:*:node.js:*:*",
+      "whereis": "cpe:2.3:a:whereis_project:whereis:*:*:*:*:*:node.js:*:*",
+      "whispercast": "cpe:2.3:a:whispercast_project:whispercast:*:*:*:*:*:node.js:*:*",
+      "wifey": "cpe:2.3:a:wifey_project:wifey:*:*:*:*:*:node.js:*:*",
+      "wifiscanner": "cpe:2.3:a:thingssdk:wifiscanner:*:*:*:*:*:node.js:*:*",
+      "wincred": "cpe:2.3:a:wincred_project:wincred:*:*:*:*:*:*:*:*",
+      "wind-mvc": "cpe:2.3:a:wind-mvc_project:wind-mvc:*:*:*:*:*:node.js:*:*",
+      "windows-build-tools": "cpe:2.3:a:windows-build-tools_project:windows-build-tools:*:*:*:*:*:node.js:*:*",
+      "windows-iedriver": "cpe:2.3:a:windows-iedriver_project:windows-iedriver:*:*:*:*:*:node.js:*:*",
+      "windows-latestchromedriver": "cpe:2.3:a:windows-latestchromedriver_project:windows-latestchromedriver:*:*:*:*:*:node.js:*:*",
+      "windows-selenium-chromedriver": "cpe:2.3:a:windows-selenium-chromedriver_project:windows-selenium-chromedriver:*:*:*:*:*:node.js:*:*",
+      "windows-seleniumjar": "cpe:2.3:a:windows-seleniumjar_project:windows-seleniumjar:*:*:*:*:*:node.js:*:*",
+      "windows-seleniumjar-mirror": "cpe:2.3:a:windows-seleniumjar-mirror_project:windows-seleniumjar-mirror:*:*:*:*:*:node.js:*:*",
+      "wintiwebdev": "cpe:2.3:a:wintiwebdev_project:wintiwebdev:*:*:*:*:*:node.js:*:*",
+      "wixtoolset": "cpe:2.3:a:wixtoolset_project:wixtoolset:*:*:*:*:*:node.js:*:*",
+      "word-wrap": "cpe:2.3:a:word-wrap_project:word-wrap:*:*:*:*:*:node.js:*:*",
+      "worksmith": "cpe:2.3:a:guidesmiths:worksmith:*:*:*:*:*:node.js:*:*",
+      "x-data-spreadsheet": "cpe:2.3:a:x-data-spreadsheet_project:x-data-spreadsheet:*:*:*:*:*:node.js:*:*",
+      "xd-testing": "cpe:2.3:a:xd-testing_project:xd-testing:*:*:*:*:*:node.js:*:*",
+      "xmldom": "cpe:2.3:a:xmldom_project:xmldom:*:*:*:*:*:node.js:*:*",
+      "xtalk": "cpe:2.3:a:xtalk_project:xtalk:*:*:*:*:*:node.js:*:*",
+      "y18n": "cpe:2.3:a:y18n_project:y18n:*:*:*:*:*:node.js:*:*",
+      "yargs-parser": "cpe:2.3:a:yargs:yargs-parser:*:*:*:*:*:node.js:*:*",
+      "yttivy": "cpe:2.3:a:yttivy_project:yttivy:*:*:*:*:*:node.js:*:*",
+      "yyooopack": "cpe:2.3:a:yyooopack_project:yyooopack:*:*:*:*:*:node.js:*:*",
+      "yzt": "cpe:2.3:a:yzt_project:yzt:*:*:*:*:*:node.js:*:*",
+      "zjjserver": "cpe:2.3:a:zjjserver_project:zjjserver:*:*:*:*:*:node.js:*:*",
+      "zwserver": "cpe:2.3:a:zwserver_project:zwserver:*:*:*:*:*:node.js:*:*"
+    },
+    "pypi": {
+      "0.0.1": "cpe:2.3:a:pypi:pypi:*:*:*:*:*:*:*:*",
```

Comment:
this seems wrong, but I haven't looked at the underlying data yet to check this (wanted to at least raise it up even if I haven't dug in yet)

Reply:
Ugh, https://nvd.nist.gov/vuln/detail/CVE-2022-34500, https://nvd.nist.gov/vuln/detail/CVE-2022-34501 - those CPEs just look really wrong

Reply:
Guessing it's because of that entry

Reply:
Yeah this was an interesting one -> https://pypi.org/project/0.0.1/